platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/EmbeddedSolrFactory.java

```diff
@@ -174,17 +172,10 @@ public static EmbeddedSolrServer getEmbeddedSolrServer(
       SolrCoreContainer container = new SolrCoreContainer(loader);
 
       String dataDirPath = null;
-      if (!ConfigurationStore.getInstance().isInMemory()) {
-        File dataDir = configProxy.getDataDirectory();
-        if (dataDir != null) {
-          dataDirPath = Paths.get(dataDir.getAbsolutePath(), coreName, "data").toString();
-          LOGGER.debug("Using data directory [{}]", dataDirPath);
-        }
-      } else {
-        PluginInfo info = solrConfig.getPluginInfo(DirectoryFactory.class.getName());
-        if (info != null && !"solr.RAMDirectoryFactory".equals(info.className)) {
-          LOGGER.debug("Using in-memory configuration without RAMDirectoryFactory.");
-        }
+      File dataDir = configProxy.getDataDirectory();
```

Comment:
❓ Not that I am not happy with this but why change the debug traces for in memory instance? It is still being used.

Reply:
This code worked correctly before we added suggestion configuration.  These changes fixed the regression.